### PR TITLE
chore: tidy cypress ts compiler messages in IDE

### PR DESCRIPTION
**Pull request by karel-rehor**
* Narrows down imports in cypress/tsconfig.json to avoid conflicts between chai and jest.  
   * Completes type declarations in support/commands.ts to remove distracting error messages
   * Additional type declarations in shared/nav.test.ts and shared/tasks.test.ts

### CODE REVIEW

**Comment by ui-ci-bot-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 57 | :red_circle:
accessibility | 96 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 100 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?lower=now%28%29%20-%202d&vars%5Blighthouse_score_branch%5D=2299)

File: package.json [hunk 1]
```diff
@@ -75,11 +75,13 @@
   "devDependencies": {
     "@babel/core": "^7.5.5",
     "@babel/preset-env": "^7.5.5",
+    "@influxdata/influxdb-client": "^1.15.0",
     "@influxdata/oats": "0.5.3",
     "@testing-library/dom": "^7.24.2",
     "@testing-library/jest-dom": "^5.11.6",
     "@testing-library/react": "^11.0.4",
     "@testing-library/user-event": "^12.1.6",
+    "@types/chai": "^4.2.21",
```
Comment (genehynson) [inline package.json:84]: any yarn.lock file changes after adding these dependencies?

**Review by genehynson — APPROVED** [review]
lgtm

**Comment by joelfried** [comment]
@karel-rehor Please rebase (or pull) master as there were changes to the tests running in our CI pipeline on master.

**Pull request merged**